src/engraving/types/symnames.cpp

```diff
@@ -5879,10 +5879,10 @@ constexpr const std::array<const char*, size_t(SymId::lastSym) + 1> SymNames::s_
     "Combining tremolo 3",
     "Combining tremolo 4",
     "Combining tremolo 5",
-    "Divide measured tremolo by 2",
-    "Divide measured tremolo by 3",
-    "Divide measured tremolo by 4",
-    "Divide measured tremolo by 6",
+    QT_TRANSLATE_NOOP("symUserNames", "Divide measured tremolo by 2"),
+    QT_TRANSLATE_NOOP("symUserNames", "Divide measured tremolo by 3"),
+    QT_TRANSLATE_NOOP("symUserNames", "Divide measured tremolo by 4"),
+    QT_TRANSLATE_NOOP("symUserNames", "Divide measured tremolo by 6"),
```

Comment:
The context should be `engraving/sym` nowadays

Reply:
Yup, done